src/eyeTracker.m

```diff
@@ -218,52 +166,54 @@
                     return
                 end
 
-                % record a few samples before we actually start displaying
-                % otherwise you may lose a few msec of data
+                % Record a few samples before we actually start displaying otherwise you may lose
+                %  a few msec of data.
                 WaitSecs(0.1);
 
-                % HERE START THE STIMULATION OF THE BLOCK
-                % to mark the beginning of the trial
+                % Mark the beginning of the trial, here start the stimulation of the experiment.
                 Eyelink('Message', 'SYNCTIME');
 
             case 'StopRecordings'
 
-                % STEP 8
-                % finish up: stop recording eye-movements,
-                % EyeLink Stop recording the block
+                %% Stop recording of eye-movements
+
+                % EyeLink Stop recording the block.
                 Eyelink('Message', 'BLANK_SCREEN');
-                % adds 100 msec of data to catch final events
+
+                % Add 100 msec of data to catch final events.
                 WaitSecs(0.1);
-                % close graphics window, close data file and shut down tracker
+
+                % Stop recoding.
                 Eyelink('StopRecording');
 
             case 'Shutdown'
 
+                %% End of the experiment
+                % Save the edf file and shut down connection with Eyelink.
+
+                % Set the edf file path + name.
                 edfFileName = fullfile( ...
```

Comment:
This needs to be updated as CPP_BIDS now follows the current BIDS version and not the one from the BEP extension.